code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -4,9 +4,18 @@
 	damage = 8
 	damage_type = TOX
 	nodamage = 0
-	knockdown = 100
-	stutter = 5
-	drowsy = 50
+	stamina = 60
+	eyeblur = 10
+	knockdown = 10
```

Comment:
Any KD at all will disarm, Even 1 KD

Reply:
As kevin explained it to me, that wasn't the case and you needed a minimum 80 for there to be a disarming effect of some kind. So I looked for alternative methods to try and force the disarm.

Reply:
wrong

Reply:
although this does need to use the override parameters as this won't work